docker/.dockerignore

```diff
@@ -0,0 +1,17 @@
+src/node_modules
```

Comment:
This file is super important and will be worth documenting in my updated `docker/README.md`.

In order to keep docker images slim and prevent accidental development artifacts from slipping it (potentially with secrets), this file instructs docker to filter which files are copied into build images similar to gitignore and npmignore.